### DISCUSSION TITLE
Confusion with Docs Spatie:Permissions

### DISCUSSION THREAD

**Discussion by SubjectXXX** [spatie/laravel-permission#2059] [Q&A] [answered]
Greetings,

in Docs of Spatie:Permissions in section about UUID, the first change you have to do is to assign 'uuid' type of column to 'id'.
But in example, both types are written.

Wouldn't it be more clear to at least comment out first type definition of unsignedBigInteger?

So instead of:

    $table->bigIncrements('id');  
    $table->uuid('id');

should be

    //$table->bigIncrements('id');    
    $table->uuid('id');

?

**Comment by erikn69** [comment, marked as answer]
yes, it's wrong, but you got the idea
